File: C++ Code/Basics/Mensuration/2 D/Rectangle.cpp
// WAP to calculate area and perimeter of a rectangle,length and breadth are given by user.
#include<iostream>
using namespace std;
int main(){
    float l,b;
    cout<<"Enter the length : ";
    cin>>l;
    cout<<"Enter the breadth : ";
    cin>>b;
    cout<<endl<<"Area = "<<l*b<<endl;
    cout<<"Perimeter = "<<2*(l+b);
    return 0;
}

File: Java Code/Basics/Mensuration/Rectangle.java
import java.util.*;

public class Rectangle{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter the length : ");
    float l=sc.nextFloat();
    System.out.print("Enter the breadth : ");
    float b=sc.nextFloat();
    System.out.println("Area of the rectangle : "+l*b);
    System.out.println("Perimeter of the rectangle  "+2*(l+b));
  }
}